src/lab_sim/objectives/take_picture_with_wrist_camera.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<root BTCPP_format="4" main_tree_to_execute="Take wrist camera picture">
+  <!--//////////-->
+  <BehaviorTree
+    ID="Take wrist camera picture"
+    _subtreeOnly="true"
+    _description="Loads wrist camera parameters and records color image."
+  >
+    <Control ID="Sequence" name="TopLevelSequence">
+      <Action
+        ID="LoadObjectiveParameters"
+        config_file_name="apriltag_detection_config.yaml"
+        parameters="{wrist_camera_parameters}"
+      />
+      <Action
+        ID="GetCameraInfo"
+        topic_name="/wrist_camera/camera_info"
+        message_out="{wrist_camera_info}"
+        timeout_sec="5.000000"
+      />
+      <Action
+        ID="GetImage"
+        topic_name="/wrist_camera/color"
+        message_out="{wrist_camera_image}"
+        timeout_sec="5.000000"
+      />
+    </Control>
+  </BehaviorTree>
+  <TreeNodesModel>
+    <SubTree ID="Take wrist camera picture">
+      <inout_port name="wrist_camera_image" default="" />
+      <inout_port name="wrist_camera_info" default="" />
+      <inout_port name="wrist_camera_parameters" default="" />
```

Comment:
These should probably be updated with default field names.

Reply:
So, just write
```xml
<inout_port name="wrist_camera_image" default="{wrist_camera_image}" />
<inout_port name="wrist_camera_info" default="{wrist_camera_info}" />
<inout_port name="wrist_camera_parameters" default="{wrist_camera_parameters}" />
```
or omit the `wrist_*`?